### ISSUE TITLE
Generating statistics for one repository

### ISSUE THREAD

**Issue by ghost** [anuraghazra/github-readme-stats#1410]
How to make
generating statistics for one repository?

**Comment by jjacenty** [comment]
Yes, I've seen it
only if someone has a 100 repository?

**Comment by rickstaa** [comment]
@jjacenty I agree that a `include` tag would be a nice addition. Maybe you can rewrite your original comment into a feature request (e.g. give a more detailed explanation of the feature you had in mind).

**Comment by AbhishekPowar** [comment]
@rickstaa . 
I have added the changes required for this issue . 
Can you please review the code ?

**Comment by rickstaa** [comment]
@AbhishekPowar Thanks a lot for implementing the code for adding this new feature. I quickly checked it, and overall it looks good. There are, however, small things I would change. Maybe you can create a pull request so that @anuraghazra and I can add comments.

**Comment by AbhishekPowar** [comment]
Hi @rickstaa,
I have raised the PR #1590 as discussed. Please review and add comments

**Comment by rickstaa** [comment]
@AbhishekPowar Thanks a lot for creating the pull request. I will try to look at the pull request later this week.
